src/NzbDrone.Core.Test/TvTests/EpisodeRepositoryTests/EpisodesWhereCutoffUnmetFixture.cs

```diff
@@ -37,12 +39,25 @@ public void Setup()
                 }
             };
 
+            var langProfile = new LanguageProfile
+            {
+                Id = 1,
+                Languages = new List<ProfileLanguageItem>
```

Comment:
There seems to be a lot of the same object creation, we get a big chuck of code cut out by creating a default profile we can use for testing.